storm/src/main/storm/mesos/MesosSupervisor.java

```diff
@@ -56,17 +55,22 @@
   Map _conf;
   // Store state on port assignments arriving from MesosNimbus as task-launching requests.
   private static final TaskAssignments _taskAssignments = TaskAssignments.getInstance();
-  // What is the storm-core supervisor's view of the assigned ports?
-  AtomicReference<Set<Integer>> _supervisorViewOfAssignedPorts = new AtomicReference<Set<Integer>>(new HashSet<Integer>());
 
   public static void main(String[] args) {
-    org.apache.storm.daemon.supervisor.launch(new MesosSupervisor());
+    Map<String, Object> conf = ConfigUtils.readStormConfig();
+
+    try {
+      Supervisor supervisor = new Supervisor(conf, null, new MesosSupervisor());
+      supervisor.launch();
+    } catch (Exception e) {
+      String msg = String.format("main: Exception: %s", e.getMessage());
+      LOG.error(msg);
+      e.printStackTrace();
+    }
   }
 
   @Override
   public void assigned(Collection<Integer> ports) {
```

Comment:
Can you please add a comment before this method mentioning that it's not being called from the supervisor anymore?